var/spack/repos/builtin/packages/python/package.py

```diff
@@ -923,12 +940,15 @@ def libs(self):
         # installs them into lib64. If the user is using an externally
         # installed package, it may be in either lib or lib64, so we need
         # to ask Python where its LIBDIR is.
-        libdir = self.config_vars['LIBDIR']
+        libdir = self.config_vars["LIBDIR"]
```

Comment:
Is the change to double quotes and here and for LIBPL intentional?

Reply:
I really don't know how that happened... Don't recall making this change, must be a rebase artifact or something. I'll revert it.